<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>700</y>
     <width>281</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Запустити</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>301</width>
     <height>127</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vLayout_del_methods">
    <item>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="text">
       <string>LRU</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="text">
       <string>LFU</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_10">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="text">
       <string>FIFO</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пам'ять</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>291</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vLayout_record_methods">
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Наскрізний запис</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Зворотній запис</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>291</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gLayout_instr_settings">
    <item row="3" column="0" colspan="3">
     <widget class="QPushButton" name="pushB_generate_instr">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Згенерувати 10 запитів</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="rBut_read">
      <property name="text">
       <string>Читання</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QRadioButton" name="rBut_mixed">
      <property name="text">
       <string>В перемішку</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="rBut_load">
      <property name="text">
       <string>Запис</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="textE_list_intr"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lEdit_num_instr">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Кількість запитів:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Запит</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>291</width>
     <height>135</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="fLayout_mem_settings">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Розмір основної пам'яті:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lEdit_mm_size">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Розмір кеш-пам'яті:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lEdit_cache_size">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Розмір 1 слова:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lEdit_word_size">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Офсет:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lEdit_offset">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Метод запису</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Метод видалення</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
